rmgpy/reaction.py

```diff
@@ -156,7 +156,19 @@ def __str__(self):
             return arrow.join([' + '.join([str(s) for s in self.reactants])+' (+'+str(self.specificCollider)+')', ' + '.join([str(s) for s in self.products])+' (+'+str(self.specificCollider)+')'])
         else:
             return arrow.join([' + '.join([str(s) for s in self.reactants]), ' + '.join([str(s) for s in self.products])])
-
+    
+    def toLabeledStr(self):
+        """
+        the same as __str__ except that the labels are assumed to exist and used for reactant and products rather than 
+        the labels plus the index in parentheses
+        """
+        arrow = ' <=> '
+        if not self.reversible: arrow = ' => '
+        if self.specificCollider:
+            return arrow.join([' + '.join([s.label for s in self.reactants])+' (+'+str(self.specificCollider)+')', ' + '.join([s.label for s in self.products])+' (+'+str(self.specificCollider)+')'])
+        else:
+            return arrow.join([' + '.join([s.label for s in self.reactants]), ' + '.join([s.label for s in self.products])])
+        
```

Comment:
Do you know if it is possible to add an option to this method to have the index in parenthesis? That way we can just have 

```def __str__(self):
    return toLabeledStr(use_index = True)
```

This would avoid code duplication, so modifications are more robust (for example someone only adds a modfication of the specific collider to __str__ and not toLabeledStr)